src/PhpPresentation/Writer/PowerPoint2007/PptCharts.php

```diff
@@ -828,6 +828,16 @@ protected function writeTypeBar(XMLWriter $objWriter, Bar $subject, $includeShee
             $this->writeSingleValueOrReference($objWriter, $includeSheet, $series->getTitle(), $coords);
             $objWriter->endElement();
 
+            // c:spPr
+            if ($series->getFill()->getFillType() != Fill::FILL_NONE) {
+                // c:spPr
+                $objWriter->startElement('c:spPr');
+                // Write fill
+                $this->writeFill($objWriter, $series->getFill());
+                // ## c:spPr
+                $objWriter->endElement();
+            }
+
```

Comment:
Just move block seems validate by xsd

Reply:
But when you generate a chart with fill, is it well displayed ?

Reply:
I can’t clearly say if it was that modification or change overlap to 100 not 10000.
But office365 website blank 2 bar chart for example say in issues 382.
I take latest screenshot in issue 382 and try to fix warnings/errors.

Why do you have an error ? I do modification ?

Reply:
Now office 365 display all charts